doc/reference/kernel/threads/workqueue.rst

```diff
@@ -122,6 +122,41 @@ queue, unless the work item has already been removed and processed by the
 workqueue's thread. Consequently, once a work item's timeout has expired
 the work item is always processed by the workqueue and cannot be canceled.
 
+Triggered Work
+**************
+
+The :cpp:func:`k_work_poll()` interface schedules a triggered work
```

Comment:
This needs to be `k_work_poll_submit()` for the documentation link to work.

Reply:
Fixed